crates/sui-framework/sources/governance/sui_system.move

```diff
@@ -79,6 +79,13 @@ module sui::sui_system {
         epoch_start_timestamp_ms: u64,
     }
 
+    struct SuiSystemStateWrapper has key {
```

Comment:
Calling this "wrapper" might be confusing to users of this module, who shouldn't need to know about the wrapping we are doing here. What if we call the wrapper "SuiSystemState" and the actual content "SuiSystemStateInner"? No strong preference though.

Reply:
Good question. I am not sure. 
My consideration is that dapps (explorer, wallet and etc) should be getting SuiSystemState for clarity.

Reply:
Let me think about that more and maybe rename it in a separate PR because that would probably lead to a whole lot of changes everywhere.

Reply:
+1 to @emmazzz's naming suggestion, as per the offline discussion!

Reply:
FWIW, I think the size of this PR will go down quite a bit if you do the rename proactively, rather than in a separate PR -- no strong compulsion on my side to do it that way though.

Reply:
hmm are you suggesting that we make Rust data structure name different from Move names?